webapp/components/ContributionForm/ContributionForm.spec.js

```diff
@@ -241,6 +242,13 @@ describe('ContributionForm.vue', () => {
           expect(mocks.$apollo.mutate).toHaveBeenCalledTimes(1)
         })
 
+        describe('questionable images should be blurred', () => {
+          it('questionable images unset be blurred', async () => {
+            await wrapper.find('input[type="checkbox"]').trigger('click')
+            expect(wrapper.find('input[type="checkbox"]').exists()).toBe(true)
+          })
+        })
+
```

Comment:
what is this even testing? you are finding an input of type checkbox, clicking on it and expecting that the element exists?? what am I missing?
It must exist, otherwise it would throw an error `Cannot trigger a click event on an empty Wrapper`, or something...
what happens when it's clicked? I guess it adds a class? so could we find the image and, `expect(img.classes()).toContain('blur-image')` or something similar?

Reply:
changed